example/lib/peripheral_details/peripheral_details_bloc.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:async';
+import 'package:blemulator_example/adapter/ble_adapter.dart';
+import 'package:blemulator_example/model/ble_peripheral.dart';
+import 'package:bloc/bloc.dart';
+import './bloc.dart';
+
+class PeripheralDetailsBloc
+    extends Bloc<PeripheralDetailsEvent, PeripheralDetailsState> {
+  BleAdapter _bleAdapter;
+  final BlePeripheral _initialPeripheral;
```

Comment:
It is final and this screen handles only one peripheral. Do you predict a possibility to change the chosen peripheral from this screen? If not, then perhaps it shouldn't be called `initial`?

Reply:
Renamed `_initalPeripheral` to `_chosenPeripheral`